aspnetcore/blazor/tooling.md

```diff
@@ -74,7 +74,7 @@ There's no centralized way to trust a certificate on Linux. Typically, one of th
 * Trust all self-signed certificates for `localhost`.
 * Add the certificate to the list of trusted certificates in the browser.
 
-For more information, see the guidance provided by your browser and Linux distro.
+For more information, see the guidance provided by your browser and Linux distribution.
```

Comment:
There are four more of these across the repo.

"distro" is a valid abbreviation of "distribution." It's used *everywhere* by just about *everyone*. There's no confusion. Maintainers use it to describe their own products. Why are we trying to force the word "distribution" in these docs?

Reply:
Nevermind ... I was just curious.

Reply:
You're correct that most folks would say "distro", including myself. The problem is localization. This term would be considered slang/jargon.

Reply:
Gotcha ... that makes sense. Thanks.

Reply:
I agree. I'll replace these with another PR right now.